test/parallel/test-fs-stat-bigint.js

```diff
@@ -10,13 +10,18 @@ const { isDate } = require('util').types;
 
 tmpdir.refresh();
 
-const fn = path.join(tmpdir.path, 'test-file');
-fs.writeFileSync(fn, 'test');
+let testIndex = 0;
 
-let link;
-if (!common.isWindows) {
-  link = path.join(tmpdir.path, 'symbolic-link');
-  fs.symlinkSync(fn, link);
+function getFilename() {
+  const filename = path.join(tmpdir.path, `test-file-${++testIndex}`);
+  fs.writeFileSync(filename, 'test');
+  return filename;
+}
+
+function linkToFile(filename) {
+  const link = path.join(tmpdir.path, `symbolic-link-${testIndex}`);
```

Comment:
Should this maybe also increment the index?

Reply:
That wouldn't hurt but also isn't necessary.

Reply:
Maybe another fun option might be to change it from:

```js
`symbolic-link-${testIndex}`
```
...to: 

```js
`${filename}-link`
```

Reply:
...which might make the whole function unnecessary? All calls to the function could be replaced with:

```js
fs.symlinkSync(filename, `${filename}-link`);
```